pkg/controller/releasemanager/plan/syncCanary.go

```diff
@@ -0,0 +1,101 @@
+package plan
+
+import (
+	"context"
+	"fmt"
+
+	monitoringv1 "github.com/coreos/prometheus-operator/pkg/apis/monitoring/v1"
+	"github.com/go-logr/logr"
+	picchuv1alpha1 "go.medium.engineering/picchu/pkg/apis/picchu/v1alpha1"
+	"go.medium.engineering/picchu/pkg/plan"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	DefaultCanaryLabel      = "canary"
+	DefaultCanaryRulesName  = "picchu.canary.rules"
+	DefaultCanaryAlertAfter = "5m"
+	canarySliFailingQuery   = "100 * (1 - %s / %s) + %f < %f"
```

Comment:
are there any cases where equality operator should be `>`?

Reply:
I couldn't think of any, since I only expose an `ErrorQuery` and `TotalQuery` for SLI configuration its (intentionally) rigid.